dt-assets/scss/_main.scss

```diff
@@ -672,6 +668,10 @@ a.dt-green:hover {
     }
 }
 
+div.button-group {
```

Comment:
What is this edit for? It applies to the whole site

Reply:
It is there to fix the spacing between the All and Unread buttons, and can prevent the same thing happening elsewhere.
Before:
<img width="812" alt="Screenshot 2019-11-26 11 33 43" src="https://user-images.githubusercontent.com/43966676/69657854-b80c8100-1040-11ea-84a3-a91fe34089c2.png">
After:
<img width="844" alt="Screenshot 2019-11-26 11 36 15" src="https://user-images.githubusercontent.com/43966676/69658019-0457c100-1041-11ea-8e7d-79d9dec53f7c.png">

Reply:
The same styling could be used to fix these grouped buttons too as they are touching. 
IMO Removing the inline style (that is coded into the html) and moving it to the css file would be an option.

<img width="1198" alt="Screenshot 2019-11-26 11 45 30" src="https://user-images.githubusercontent.com/43966676/69658752-65cc5f80-1042-11ea-9d6c-d1b16b72486d.png">

Reply:
The button-group class is used in several places in the site (ie milestones). Each place will need it's own css.
For the notification buttons please use specific css instead of general.

Reply:
> The button-group class is used in several places in the site (ie milestones). Each place will need it's own css.
> For the notification buttons please use specific css instead of general.

Ok. So do you suggest I write CSS to be location specific - just to be used on the notification page? So adjust the CSS file would be the cleanest IMO.

Reply:
The scss code you wrote applies to tho whole theme, maybe in ways you won't expect.
Either you change it to work for every situation, or you make your selectors more specific.